probe/cobra.go

```diff
@@ -0,0 +1,35 @@
+package probe
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func CobraCommand() *cobra.Command {
+	var (
+		probeOptions Options
+	)
+
+	prb := &cobra.Command{
+		Use:   "probe",
+		Short: "Check the liveness or readiness of a locally-running server",
```

Comment:
Can you add an `Example:` section?  Some of the options, like `probe-path`, are not clear to me.

Reply:
Done.